bindings/nodejs/generated.d.ts

```diff
@@ -32,6 +32,16 @@ export interface ListOptions {
   limit?: number
   recursive?: boolean
 }
+export interface WriteWith {
```

Comment:
I feel like this API should be called as `WriteOptions`. What do you think?

Reply:
sound reasonable, I will change it and `WriterOptions`